tests/unit/s2n_drbg_test.c

```diff
@@ -233,20 +383,23 @@ int main(int argc, char **argv)
     }
     EXPECT_SUCCESS(s2n_timer_reset(config, &timer, &urandom_nanoseconds));
 
+    printf("\nTime to generate 32 MB of random data per DRBG in nanoseconds\nAES 128 CTR: %"PRId64 "\nAES 256 CTR: %" PRId64" \nurandom:     %" PRId64" \n",
```

Comment:
I don't think we want this printf statement to be checked in.

Reply:
Removed it.